packages/contracts-bedrock/test/L1/DataAvailabilityChallenge.t.sol

```diff
@@ -321,6 +496,19 @@ contract DataAvailabilityChallenge_Challenge_Test is DataAvailabilityChallenge_T
         dataAvailabilityChallenge.challenge(0, computeCommitmentKeccak256("some other hash"));
     }
 
+    /// @notice Test that the `challenge` function reverts at challenge window boundary.
+    function test_challenge_challengeWindowBoundary_reverts() public {
```

Comment:
AFAICT this test doesn't do anything different from this one:

https://github.com/ethereum-optimism/optimism/blob/0e0cf04c3e8bb579823ecc04c87efea50c83d1e7/packages/contracts-bedrock/test/L1/DataAvailabilityChallenge.t.sol#L530-L541